src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,187 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+
+    private static User user;
+    private static RegistrationServiceImpl registrationService;
+
+    private static final int DEFAULT_VALID_AGE = 20;
+    private static final String DEFAULT_VALID_LOGIN = "some_login";
+    private static final String DEFAULT_VALID_PASSWORD = "12345678";
+    private static final int INVALID_AGE = 15;
+    private static final String INVALID_LOGIN = "log";
+    private static final String INVALID_PASSWORD = "123";
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setAge(DEFAULT_VALID_AGE);
+        user.setLogin(DEFAULT_VALID_LOGIN);
+        user.setPassword(DEFAULT_VALID_PASSWORD);
+    }
+
+    @Test
+    void register_validUser_Ok() {
+        registrationService.register(user);
+        assertTrue(Storage.people.contains(user), "Valid user should be registered");
+    }
+
+    @Test
+    void register_nullValue_NotOk() {

```

Comment:
```suggestion
    void register_nullValue_notOk() {
```
please fix the same issue in all places